ui/guest/src/store/epics/root.ts

```diff
@@ -152,7 +151,7 @@ const epic = combineEpics<GuestStandardAction, GuestStandardAction, GuestState>(
             e.dataTransfer.setData('text/plain', `${record.id}`);
             e.dataTransfer.setDragImage(document.querySelector('.craftercms-dragged-element'), 20, 20);
           }
-          $('html').addClass(dragAndDropActiveClass);
+          document.querySelector('html').classList.add(dragAndDropActiveClass);
```

Comment:
Perhaps easier/more performant to do `document.documentElement`. Change all that apply. Similarly, if you're querying for the body somewhere, use `document.body`.